teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -545,6 +555,21 @@ interface Investigation {
     val scope: InvestigationScope
 }
 
+interface Investigation : InvestigationMuteBase {
+    val state: InvestigationState
+}
+
+interface Mute : InvestigationMuteBase {
+    val tests: List<Test>?
+}
+
+/**
+ * Data class to store Test data, to be replaced with API to handle requests to the tests
+ */
+data class Test(val id: String, val name: String) {
```

Comment:
This feels half-cooked. Is it really enough to have a Test without test runs?
In any case, let's use TestId type for id.

Reply:
I've added interface to handle app/rest/tests. As for TestRuns, those are not listed in the mutes when requested, so those can be then processed using buildID, etc. API for the mutes only lists currently existing ones, and do not show state in the given build. For our use case, we only need an id and name. TestRun api is already there.
Could you please take a look. If this is alright now? We can then extend it to return testOccurences field, for instance, as per documentation [here](https://www.jetbrains.com/help/teamcity/rest/test.html#Schema)